pauga family reunion/pauga family reunion/UserService/UserService.swift

```diff
@@ -36,6 +37,10 @@ class UserService : UserServicable {
         setCurrentUser(userLoginModel: userLoginModel)
     }
     
+    func signIn(email: String, password: String) async throws {
+        // signIn here
```

Comment:
The sign in endpoint is finished so let's go ahead and add the code to call it, handle errors and set the user